core/src/main/java/io/th0rgal/oraxen/utils/tuneblock/TuneBlock.java

```diff
@@ -0,0 +1,97 @@
+package io.th0rgal.oraxen.utils.tuneblock;
```

Comment:
this really doesnt need to be its own class. Atleast move it into noteblock mechanic package and rename to RegularNoteBlock or something